Rakefile

```diff
@@ -73,7 +73,7 @@ namespace :db do
   end
 
   desc "Migrate database up to latest migration available"
-  task :migrate do
+  task :migrations do
```

Comment:
Why did you do this?

Reply:
This wasn't intentional. What I indeed renamed intentional was the migrate -> migration folder. Apparently this had the side effect of modifying the Rakefile. The reason of the folder rename is just that it expresses better the intention.